src/cljs/main/broadfcui/common/components.cljs

```diff
@@ -325,10 +325,9 @@
                 (when (seq stack-trace)
                   [StackTraceViewer {:lines stack-trace}])]]
               [:div {}
-               (style/create-link {:text [:span {}
-                                          (icons/icon {:className "fa-fw"} :disclosure-closed)
-                                          "Show Details"]
-                                   :onClick #(swap! state assoc :expanded? true)})])]))))})
+               (links/create-internal {:onClick #(swap! state assoc :expanded? true)}
```

Comment:
Should we have people change the indent for these so they use normal indent for next line, like creating an element?